apps/music/index.html

```diff
@@ -95,7 +95,7 @@ <h1 id="title-text"><bdi data-l10n-id="music"></bdi></h1>
       </div>
       <div id="views-player" class="view hidden">
         <div id="player-cover">
-          <div id="player-cover-image"></div>
+          <img id="player-cover-image"></img>
```

Comment:
Thanks Jim, I forget we don't need to close a <img>. But because of https://bugzilla.mozilla.org/show_bug.cgi?id=1168549#c14, I will keep the <div>(background-image) here, I cannot get rid of the flicker from "object-fit: cover" and I don't want to use some workaround, since the player view has no problem here. A followup would be nice so I will file one to replace the div to img for the player view(for master only I think).